src/IB/IBFEMethod.cpp

```diff
@@ -387,10 +811,42 @@ IBFEMethod::registerStressNormalizationPart(unsigned int part)
     if (d_is_stress_normalization_part[part]) return;
     d_has_stress_normalization_parts = true;
     d_is_stress_normalization_part[part] = true;
-    System& Phi_system = d_equation_systems[part]->add_system<LinearImplicitSystem>(PHI_SYSTEM_NAME);
-    d_equation_systems[part]->parameters.set<Real>("Phi_epsilon") = d_epsilon;
-    Phi_system.attach_assemble_function(assemble_poisson);
-    Phi_system.add_variable("Phi", d_fe_order[part], d_fe_family[part]);
+    System& Phi_system = d_equation_systems[part]->add_system<TransientLinearImplicitSystem>(PHI_SYSTEM_NAME);
+    d_equation_systems[part]->parameters.set<Real>("Phi_epsilon") = d_phi_epsilon;
+    d_equation_systems[part]->parameters.set<Real>("ipdg_jump0_penalty") = d_ipdg_jump0_penalty;
+    d_equation_systems[part]->parameters.set<Real>("ipdg_jump1_penalty") = d_ipdg_jump1_penalty;
+    d_equation_systems[part]->parameters.set<Real>("ipdg_beta0") = d_ipdg_beta0;
+    d_equation_systems[part]->parameters.set<Real>("ipdg_beta1") = d_ipdg_beta1;
+    d_equation_systems[part]->parameters.set<Real>("cg_penalty") = d_cg_penalty;
+    d_equation_systems[part]->parameters.set<std::string>("Phi_solver") = d_phi_solver;
+    d_equation_systems[part]->parameters.set<Real>("dt") = d_phi_dt;
+    d_equation_systems[part]->parameters.set<Real>("Phi_diffusion") = d_phi_diffusion;
+
+    // assign function for building Phi linear system.  defaults to CG discretization
+    if (d_phi_solver.compare("CG") == 0)
+    {
```

Comment:
Also, each of these strings ("CG", "IPDG", etc) probably should be a constant static member variable, as is done for other "distinguished strings" such as the system names